bin/mfe_dev

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ $# -eq 0 ]
+  then
+    echo "Please provide a MFE name. Available MFE:"
+    echo
+    for mfe in $(ls mfe | grep frontend-app- )
+    do
+        echo ${mfe/#"frontend-app-"}
+    done
+    exit 1
+fi
+
+MFE_DIRECTORY="./mfe/frontend-app-$1"
```

Comment:
We may probably want to improve this by using the `TUTOR_ROOT` environment variable to build an absolute path to the mfe directories